testing/test_feature_store.py

```diff
@@ -66,43 +66,43 @@ def test_initialized(self, store):
     def test_get_existing_feature(self, store):
         store = self.base_initialized_store(store)
         expected = self.make_feature('foo', 10)
-        assert store.get('foo') == expected
+        assert store.get('foo', lambda x: x) == expected
```

Comment:
It might be clearer to define this identity function once, and reference it in all the places.